google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/FlexibleFacetEditor.java

```diff
@@ -78,6 +80,21 @@
   private JPanel dockerfilePanel;
   private AppEngineFlexibleFacetConfiguration facetConfiguration;
 
+  public enum ConfigFile {
```

Comment:
just curious, you think having this enum is better than passing in the callback like it was before? I like the callback because it gets rid of boilerplate.